workshop/controllers/ui/components/autoSave.lua

```diff
@@ -0,0 +1,19 @@
+-- Auto Save 
+local shared = require("tevgit:workshop/controllers/shared.lua")
+local autoSave = {}
+
+autoSave.Enabled = false
+autoSave.timeLimit = 5 -- Default (move to settings value soon)
+
+autoSave.Sync = function()
+    return spawnThread(function()
+        while true do
+            if autoSave.Enabled then
+                shared.workshop:saveGame()
+            end
+            wait(autoSave.timeLimit)
+        end
+    end)
+end
+
+return autoSave
```

Comment:
As with format, just leave an extra line at the end.
![image](https://user-images.githubusercontent.com/44125644/72883702-8407c080-3d50-11ea-91d3-770cd463fbc1.png)

Reply:
Might just be my OCD, but like yeah